tests/nn/data_parallel/test_sharded_ddp_pytorch_parity.py

```diff
@@ -183,7 +200,9 @@ def closure_sharded(input_tensor=input_tensor):
 @pytest.mark.parametrize("grad_accumulation", [True, False])
 @pytest.mark.parametrize("change_train_graph", [True, False])
 @pytest.mark.parametrize("fp16_reduction", _test_fp16_reduction)
-def test_ddp_parity(reduce_buffer_size, grad_accumulation, change_train_graph, fp16_reduction):
+@pytest.mark.parametrize("clip_grad_norm", [True, False])
```

Comment:
this is the PR: add a clip_grad_norm test, clean up the AMP tests (passed as a parameter from the outside, torch/cuda version permitting)